Assignment/Homework_1 - HTML/Pokedex_reema.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>Pokedex</title>
+<link rel="stylesheet" type="text/css" href="stylesheet-pokedex.css">
+
+</head>
+
+<body background="assets/backgrounds/background_home.png" style="border:2px solid black; width: 470px;">
+
+
+<header style="background-color: #bd5858">
+	<button type="button" onclick="alert('Tadaaa!')">Dropdown</button>
+	<span style= "color:white; padding: 135px;">Pokedex</span>
+	<img src="assets/icons/pokeball_icon.png" style="width:40px;height:40px;">
+</header>
+
+
+<section id="main">
+	<input type="text" size="70" name="search-pokemon" value="Search Pokemon" style="padding: 10px 10px 10px 10px; border:1px solid black; margin:10px;">
```

Comment:
you can try search for the difference between using name versus id! it'll be useful later.